assets/src/util/util.tsx

```diff
@@ -51,16 +82,20 @@ export const getScreenSide = (): ScreenSide | null => {
   return isScreenSide(screenSide) ? screenSide : null;
 };
 
-const isRotationIndex = (value: string | undefined) => {
-  return value === "0" || value === "1" || value === "2";
-};
-
-export const getRotationIndex = () => {
-  const rotationIndex = isDup()
+export const getRotationIndex = (): RotationIndex | null => {
+  const rotationIndex = isOFM()
     ? ROTATION_INDEX.toString()
     : getDatasetValue("rotationIndex");
 
   return isRotationIndex(rotationIndex) ? rotationIndex : null;
 };
 
+export const getTriptychPane = (): TriptychPane | null => {
```

Comment:
So the big difference between DUPs and Triptychs is that each "pane" is a unique `playerName`. So I don't think this will be as reliable for triptychs. We may need to add the pane to the screen's config. In the CSV they gave us, we can see which `playerName` is for which pane.

Reply:
Ahhhhh shoot. Ok. Will need to rethink this.